src/pages/organizations/[id]/index.js

```diff
@@ -367,7 +399,29 @@ class Organization extends Component {
           <Box layerStyle={'shadowed'} as='section'>
             <Heading variant='sectionHead'>Teams</Heading>
             {this.renderMap(teams)}
-            <TeamsTable type='org-teams' orgId={org.data.id} />
+            <Checkbox
+              border={'2px'}
+              marginTop={'-4rem'}
+              marginLeft={'1rem'}
+              position='absolute'
+              zIndex='2000'
+              borderColor='brand.600'
+              p={2}
+              bg='white'
+              name='map-bounds-filter'
+              id='map-bounds-filter'
+              type='checkbox'
+              colorScheme={'brand'}
+              isChecked={searchOnMapMove}
+              onChange={(e) => this.setSearchOnMapMove(e)}
+            >
+              Filter teams by map
+            </Checkbox>
```

Comment:
```suggestion
            {this.renderMap(teams, searchOnMapMove)}
```
Currently the "filter teams by map" checkbox renders before the map, which looks strange. Suggest including the checkbox in the renderMap function so that it only renders with the map.